script/cileaks.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -x
+
+# If this platform doesn't support test execution, bail out now
+if [ -n "$SKIP_TESTS" ];
+then
+	exit $?;
```

Comment:
Why do you pass back the value of the previous test? Shouldn't this just exit success?

Reply:
This is a vanilla copy-paste of the block just above, which appeared in [8f426d7](https://github.com/libgit2/libgit2/commit/8f426d7dd26856150950d080fb821044bb2ea811#diff-9e7fd40b8d82d14b32f15f7c2b6eb485R16). I honestly have no idea why it's written that way, but it seems to predate AppVeyor, so there's that.

Reply:
True. Let's leave it as-is, then. But I'd be grateful if you at least remove these useless semicolons ;)